### DISCUSSION TITLE
CSS themes

### DISCUSSION THREAD

**Discussion by throughamad** [pear-devs/pear-desktop#1144] [Q&A]
Hi guys, 

I have been searching for CSS themes to download and import to the app, but couldn't find any. 

Can anyone tell me where can I find and download themes from?

**Comment by Ugau943** [comment]
@reHamad 
> Can anyone tell me where can I find and download themes from?

Yes, here it is #1179 and https://github.com/dracula/dracula-theme/issues/872
